paddle/fluid/operators/collective/allreduce_op.cc

```diff
@@ -20,7 +20,7 @@ limitations under the License. */
 namespace paddle {
 namespace operators {
 
-class AllReduceOp : public framework::OperatorWithKernel {
+class AllReduceDelOp : public framework::OperatorWithKernel {
```

Comment:
what is the meaning of the `Del` in the AllReduceDelOp?

Reply:
> what is the meaning of the `Del` in the AllReduceDelOp?

because of conflicts, this op is discussed to be removed, unused now but need to be verified